@uportal/sidebar-nav/src/assets/linkData.json

```diff
@@ -0,0 +1,180 @@
+{
```

Comment:
can this be loaded from the assets folder? would it make sense to move it to the `public` folder where it can be url loaded?